lib/app/connector.rb

```diff
@@ -28,6 +28,8 @@ def start!
 
       def pre_flight_check
         raise "#{App::Config['service_type']} is not a supported connector" unless Connectors::REGISTRY.registered?(App::Config['service_type'])
+        Framework::ElasticConnectorActions.ensure_connectors_index_exists
```

Comment:
This won't stop the app from exiting as there won't be any entry in the connector index.